src/main/kotlin/com/example/mylotto/view/ResultView.kt

```diff
@@ -0,0 +1,38 @@
+package com.example.mylotto.view
+
+import com.example.mylotto.enum.Rank
+import com.example.mylotto.model.LottoResult
+import com.example.mylotto.model.LottoTicket
+
+class ResultView {
+    fun displayPurchasedTickets(tickets: List<LottoTicket>) {
+        println("${tickets.size}개를 구매했습니다.")
+        tickets.forEach { ticket ->
+            println(ticket.numbers.joinToString(prefix = "[", postfix = "]") { it.number.toString() })
+        }
+
+        println()
+    }
+
+    fun displayWinningStatistics(result: LottoResult) {
+        println()
+        println("당첨 통계")
+        println("---------")
+
+        Rank.entries
+            .filter { it != Rank.MISS }
+            .sortedBy { it.winningMoney }
+            .forEach { rank ->
+                val count = result.rankCountMap.getOrDefault(rank, 0)
+                println("${rank.countOfMatch}개 일치 (${rank.winningMoney}원)- ${count}개")
```

Comment:
```suggestion
                println("${rank.countOfMatch}개 일치 (${rank.winningMoney}원)- $count 개")
```

단순 변수일 경우 문자열 템플릿은 요렇게도 사용할 수 있습니다!

Reply:
감사합니다!

저는 아무래도 숫자랑 "개"를 붙이고 싶어져서 중괄호를 유지하려고 합니다...!